manifest.json

```diff
@@ -1,290 +1,301 @@
-        "tarRootDirPrefix": "GoogleAppEngineMapReduce-",
-        "rootDirPrefix": "gae-mapreduce-",
-        "targetDirPrefix": "gae-mapreduce-"
-      },
-      "gaePipeline": {
-        "version": "1.9.17.0",
-        "tarUrl": "https://pypi.python.org/packages/source/G/GoogleAppEnginePipeline/GoogleAppEnginePipeline-1.9.17.0.tar.gz#md5=9fe87b281f4b0a7c110534df4e61b6ec",
-        "tarRootDirPrefix": "GoogleAppEnginePipeline-",
-        "rootDirPrefix": "gae-pipeline-",
-        "targetDirPrefix": "gae-pipeline-"
-      },
-      "graphy": {
-        "version": "1.0.0",
-        "tarUrl": "https://pypi.python.org/packages/source/G/Graphy/Graphy-1.0.0.tar.gz#md5=390b4f9194d81d0590abac90c8b717e0",
-        "tarRootDirPrefix": "Graphy-",
-        "rootDirPrefix": "graphy-",
-        "targetDirPrefix": "graphy-"
-      },
-      "html5Lib": {
-        "version": "0.95",
-        "zipUrl": "https://github.com/html5lib/html5lib-python/archive/0.95.zip",
-        "rootDirPrefix": "html5lib-python-",
-        "targetDirPrefix": "html5lib-python-"
-      },
-      "simpleJson": {
-        "version": "3.7.1",
-        "tarUrl": "https://pypi.python.org/packages/source/s/simplejson/simplejson-3.7.1.tar.gz#md5=c76c2d11b87e9fb501bd0b2b72091653",
-        "tarRootDirPrefix": "simplejson-",
-        "rootDirPrefix": "simplejson-",
-        "targetDirPrefix": "simplejson-"
-      }
-    },
-    "frontend": {
-      "angular": {
-        "version": "1.4.7",
-        "srcUrl": "https://ajax.googleapis.com/ajax/libs/angularjs/1.4.7",
-        "targetDirPrefix": "angularjs-",
-        "files": [
-          "angular.js", "angular.min.js", "angular.min.js.map", "angular-animate.js",
-          "angular-animate.min.js", "angular-animate.min.js.map", "angular-resource.js",
-          "angular-resource.min.js", "angular-resource.min.js.map", "angular-route.js",
-          "angular-route.min.js", "angular-route.min.js.map", "angular-sanitize.js",
-          "angular-sanitize.min.js", "angular-sanitize.min.js.map", "angular-aria.js",
-          "angular-aria.min.js", "angular-aria.min.js.map"
-        ],
-        "bundle": {
-          "js": [
-            "angular-animate.js", "angular-resource.js", "angular-sanitize.js",
-            "angular-aria.js"
-          ]
-        }
-      },
-      "angularTest": {
-        "version": "1.4.7",
-        "srcUrl": "https://code.angularjs.org/1.4.7",
-        "targetDirPrefix": "angularjs-",
-        "files": ["angular-mocks.js", "angular-scenario.js"]
-      },
-      "bootstrap": {
-        "version": "3.3.4",
-        "zipUrl": "https://github.com/twbs/bootstrap/releases/download/v3.3.4/bootstrap-3.3.4-dist.zip",
-        "rootDir": "bootstrap-3.3.4-dist",
-        "targetDirPrefix": "bootstrap-",
-        "bundle": {
-          "css": ["css/bootstrap.css"],
-          "fontsPath": "fonts",
-          "js": ["js/bootstrap.js"]
+      "zip": {
```

Comment:
I actually find this a bit unintuitive, and would lean more towards having a "type" key for the object describing each dependency.

The reason is as follows: let's say I want to look up the html5Lib dependency. I would expect to look at backend > html5Lib; I wouldn't remember whether it was downloaded as tar or zip, which seems like a lower-level detail.

What are your thoughts?